Maquetado/scss/detail.scss

```diff
@@ -36,6 +36,14 @@
   height: 385px;
   padding: 20px;
 }
+.bookBadge{
+  position: absolute;
+  width: 55px;
+  height: 85px;
+  transform: rotate(20deg);
+  margin-top: 8px;
+  left: 14.5%;
```

Comment:
This value in percentage could affect the position of teh badge on responsive views.